### DISCUSSION TITLE
get random fake static image url from api

### DISCUSSION THREAD

**Discussion by rosnk** [faker-js/faker#1143] [Q&A]
I want fake image url to seed database, something like 
`https://loremflickr.com/cache/resized/65535_52183986434_128d52269d_h_673_1280_nofilter.jpg` which always gives same image. 

However on using following api 
`faker.image.cats(1234, 2345) `
which gives following url which is saved in db. 
`https://loremflickr.com/1234/2345/cats`

this url always gives different cat images while used in application. 

Problem: 
Can i some how run the above request and get static url and save that image url OR
is there any api call i can do to get random static image.

**Comment by rosnk** [comment]
managed to do this way, i wander if there is more elegent way to do this: 

const res_data = await axios.get(random_img_url);
const img_url = res_data.request.res.responseUrl,

**Comment by ST-DDT** [comment]
Please upvote the following issue:
https://github.com/faker-js/faker/issues/487
